modules/log6x-supported-log-outputs.adoc

```diff
@@ -0,0 +1,97 @@
+// Module included in the following assemblies:
+//
+// * observability/logging/logging-6.0/log6x-output.adoc
```

Comment:
This is outdated, and definitely should not mention fluentd.

Reply:
This seems to be outdated now. Would you like to make those updates as part of our larger PR? Please let me know if you have other thoughts. Thanks!

Reply:
All references to fluentd can be removed for 6.0

Reply:
Reference to fluent forward can be dropped as well since it is not supported in 6.0